app/views/records/edit_fields/_date_coverage.html.erb

```diff
@@ -1,24 +1,78 @@
-<% @wmon = (1..12).to_a %>
-<% @wday = (1..31).to_a %>
-Date Coverage
-<% if f.object.class.multiple? key %> 
-
-FROM
- <%= f.input :YYYY1, as: :extra, input_html: { :style=> 'width: 100px, class: form-horizontal'} %>/
-<%= select_tag :MM1, options_for_select(@wmon) %>
-<%= select_tag :DD1, options_for_select(@wday) %> 
- TO 
-<%= f.input :YYYY2, as: :extra, input_html: { :style=> 'width: 100px, class: form-horizontal'} %>/
-<%= select_tag :MM2, options_for_select(@wmon) %>
-<%= select_tag :DD2, options_for_select(@wday) %>
-<% else %>
-
-FROM
-<%= f.input :YYYY1, as: :extra, input_html: { :style=> 'width: 100px, class: form-horizontal'} %>/
-<%= select_tag :MM1, options_for_select(@wmon) %>
-<%= select_tag :DD1, options_for_select(@wday) %>
- TO 
-<%= f.input :YYYY2, as: :extra, input_html: { :style=> 'width: 100px, class: form-horizontal'} %>/
-<%= select_tag :MM2, options_for_select(@wmon) %>
-<%= select_tag :DD2, options_for_select(@wday) %>
-<% end %>
+<% @wmon = (1..12).to_a.unshift('--') %>
+<% @wday = (1..31).to_a.unshift('--') %>
+
+<div class="form-group <% if f.object.class.multiple? key %>multi_value<% end %>">
+
+  <p class="text-bold">Date Coverage</p>
+  <p class="help-block">The date range the work's data was created, as determined by the creator.</p>
+
+  <% if f.object.class.multiple? key %>
+  <ul class="listing">
+    <li class="field-wrapper">
+  <% end %> 
+
+      <div class="row">
+        <div class="col-md-6">
+          <div class="row">
+
+            <fieldset>
+              <div class="col-md-12">
+                <legend class="legend-label">Start Date</legend>
+              </div>
+
+              <div class="col-md-4">
+                <label for="YYYY1" data-label="Date">Year</label>
+                <input type="text" id="YYYY1" class="form-control date-year"></input>
```

Comment:
I don't think this input tag is getting picked up as part of the form.  Likely the form method needs to be called to generate the tag.  `<%= f.input ` or something of that nature.